src/mame/namco/namcops2.cpp

```diff
@@ -122,8 +125,8 @@ NM00053 ?
 NM00054 ?
 NM00055 ?
 
-Note about System:
-This is a general guideline flag showing compatibility between the different revisions of System 246 hardware.
+Note about :
+This is a general guideline flag showing compatibility between the different revisions of  246 hardware.
 1-7: Tested Working
```

Comment:
I wrote this documentation. No sorry, you can not remove the word System here. 'Note about:' means nothing. The notes are about "SYSTEM 246 hardware", not "246 hardware".

Reply:
Already corrected https://github.com/mamedev/mame/pull/11581/commits/1a5e1cd4d2384f5b93e7ea7fdf922572cb1dbacb

Reply:
@Gurudumps Missclicked and deleted the word System by mistake, this is **not** an attack to your work. Will correct.